src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int index = new Random().nextInt(Color.values().length);
```

Comment:
A single instance of this class will always return the same color. No need to move the index into a class variable. Take out the random class as a constant, and get the index every time in the middle of the method.